tests/app/test_app.py

```diff
@@ -67,24 +72,25 @@ def setUp(self):
 
     def test_that_checks_post_request_is_within_database(self):
         """check messages from messageSend endpoint saved in database correctly"""
-        # check if json message is inside the database
 
         url = "http://localhost:5050/message/send"
 
         data = {'msg_to': ['0a7ad740-10d5-4ecb-b7ca-3c0384afb882'],
-                'msg_from': 'ce12b958-2a5f-44f4-a6da-861e59070a31',
+                'msg_from': AppTestCase.SPECIFIC_INTERNAL_USER,
                 'subject': 'MyMessage',
                 'body': 'hello',
                 'thread': "?",
                 'ru_id': 'f1a5e99c-8edf-489a-9c72-6cabe6c387fc',
                 'create_date': datetime.now(timezone.utc),
-                'read_date': datetime.now(timezone.utc)}
+                'survey': 'a-survey-id'}
```

Comment:
Is this a uuid or a number (stocks has a survey_id of 017 for instance)?  Can we have a real looking value here?

Reply:
In use it is a uuid but is not tested here so it could be anything